geopandas/tests/test_crs.py

```diff
@@ -340,13 +338,9 @@ def test_dataframe_setitem(self):
         "scalar", [None, Point(0, 0), LineString([(0, 0), (1, 1)])]
     )
     def test_scalar(self, scalar):
-        with pytest.warns(FutureWarning):
+        with pytest.raises(ValueError, match="Assigning CRS to a GeoDataFrame without"):
             df = GeoDataFrame()
```

Comment:
Can you update this test to use a valid GeoDataFrame, and add a separate test for asserting the error? (checking that setting the crs with a scalar number works is still useful)